### ISSUE TITLE
Make a new setup 

### ISSUE THREAD

**Issue by EQuimper** [FreeCodeCampCalgary/All-About-JavaScript#7]
Maybe folder for each kind of library ....

We need to make it easier for someone to find what he need

**Comment by misterhtmlcss** [comment]
True, but putting in effort blindly and without feedback is just wasteful. If more people would use it or even try to use it and give us feedback via Issues, then we could put in the effort to shape the repository more. 

I propose @EQuimper that you ask people for feedback. Invite them to come and check it out and to open one Issue each with an eye to something they couldn't find, something they couldn't understand, etc as examples of what their problem or barrier was to success.

This then gives all of us @Dave1089 more of an understanding of where to focus e.g. more on content in certain areas, more instructions, more images, include videos, etc. 

I'm happy to do what I can, but it seems rather broad the solution to a problem that we don't fully understand. 

Thoughts?

**Comment by EQuimper** [comment]
Perfect yes true, gonna ask for.
